src/javascript/app_2/App/Components/Layout/Header/__tests__/toggle_notifications_drawer.spec.js

```diff
@@ -0,0 +1,19 @@
+import React                         from 'react';
+import { expect }                    from 'chai';
+import { configure, shallow }        from 'enzyme';
+import Adapter                       from 'enzyme-adapter-react-16';
+import { ToggleNotificationsDrawer } from '../toggle_notifications_drawer.jsx';
+import { Notifications }             from '../../../Elements/Notifications';
+
+configure({ adapter: new Adapter() });
+
+describe('ToggleNotificationsDrawer', () => {
+    it('should render one <ToggleNotificationsDrawer /> component', () => {
+        const wrapper = shallow(<ToggleNotificationsDrawer />);
+        expect(wrapper).to.have.length(1);
+    });
+    it('should have <ToggleNotificationsDrawer /> component', () => {
```

Comment:
I think function name should change to `should render <Notifications /> component`